security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -12,6 +12,17 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-22"]
+== OpenShift File Integrity Operator 0.1.22
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.22:
+
+* link:https://access.redhat.com/errata/RHBA-2022:xxxx[RHBA-2022:xxxx OpenShift File Integrity Operator Bug Fix]
```

Comment:
* link:https://access.redhat.com/errata/RHBA-2022:xxxx[RHBA-2022:0142 OpenShift File Integrity Operator Bug Fix]

Reply:
Updated to 
* link:https://access.redhat.com/errata/RHBA-2022:0142[RHBA-2022:0142 OpenShift File Integrity Operator Bug Fix]